src/mimeparser.rs

```diff
@@ -274,7 +283,30 @@ impl MimeMessage {
             is_mime_modified: false,
             decoded_data: Vec::new(),
         };
-        parser.parse_mime_recursive(context, &mail, false).await?;
+
+        match partial {
+            None => {
+                parser.parse_mime_recursive(context, &mail, false).await?;
+            }
+            Some(org_bytes) => {
+                let mut text = format!(
+                    "[{} message]",
```

Comment:
Should be translatable via stock strings?

Reply:
also here: yes, i'll change that.